### PR TITLE
Many resized versions of the new logo for android and ios

### PR DESCRIPTION
**Pull request by juanky201271**
A lot of work resizing those images, but I need them for android and ios stores.

### CODE REVIEW

File: rust/ios/Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "rustios"
 version = "1.0.1"
-authors = ["Aditya Kulkarni <aditya@zecwallet.co>", "Zingolabs <zingo@zingolabs.com>"]
```
Comment (skyl) [inline rust/ios/Cargo.toml]: Maybe best to add rather than replace @zancas ?

**Comment by skyl** [comment]
ACK pending decision whether to keep existing authors

**Comment by juanky201271** [comment]
So what about the logos? Do they seem correct...  ?

**Review by dannasessha — APPROVED** [review]
Looks good to me!

**Pull request merged**